rest-api/alembic/versions/648d99c5c414_add_authored_column.py

```diff
@@ -0,0 +1,59 @@
+"""add authored column
+
+Revision ID: 648d99c5c414
+Revises: 80d36c1e37e2
+Create Date: 2019-03-08 11:25:48.367648
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import model.utils
+
+
+from participant_enums import PhysicalMeasurementsStatus, QuestionnaireStatus, OrderStatus
+from participant_enums import WithdrawalStatus, WithdrawalReason, SuspensionStatus, QuestionnaireDefinitionStatus
+from participant_enums import EnrollmentStatus, Race, SampleStatus, OrganizationType, BiobankOrderStatus
+from participant_enums import MetricSetType, MetricsKey
+from model.site_enums import SiteStatus, EnrollingStatus, DigitalSchedulingStatus, ObsoleteStatus
+from model.code import CodeType
+
+# revision identifiers, used by Alembic.
+revision = '648d99c5c414'
+down_revision = '80d36c1e37e2'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('questionnaire_response', sa.Column('authored', model.utils.UTCDateTime(), nullable=True))
+    # ### end Alembic commands ###
+    op.execute('ALTER TABLE questionnaire_response CHANGE COLUMN `authored` `authored` DATETIME NULL AFTER created')
```

Comment:
Why we need to change it here?

Reply:
The change is setting the new field position so it is after the 'created' column instead of at the end of the table. That way it doesn't have to be after the big 'resource' column when you select from the table.